imagersite/imager_images/views.py

```diff
@@ -120,6 +109,40 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+class AlbumEditView(LoginRequiredMixin, UpdateView):
+    """
+    This creates a veiw that allows users to edit albums
+    """
+    template_name = 'imager_images/album_edit.html'
+    model = Album
+    form_class = AlbumEditForm
```

Comment:
It is not necessary to have separate forms for edit and create since the form automatically gets filled in from the instance.